report/src/introduction/agile_approach.tex

```diff
@@ -1,29 +1,27 @@
 \section{Implementing Agile Project Management with Scrum}
 Developing a software product for a business can be a complex process.
-Incremental agile approaches have become popular to address this complexity, as they allow for faster development and adaptation to changes.\cite{sommervilleSoftwareEngineering2016}
-In incremental agile approaches, a system is developed as a series of increments, each adding functionality to the system.
-Between these increments, one gathers feedback from users to prioritize functionality for the next increment.
+Incremental agile approaches have become popular to address this complexity, as it allows for fast adaptation to changes in user needs. \cite{sommervilleSoftwareEngineering2016}
+In incremental-agile approaches, system development is considered a series of increments, each adding functionality to the system.
+Between these increments, one gathers feedback from stakeholders to prioritize functionality for the next increment.
 Therefore, each increment adds value to the user.
-By continuously getting feedback and working with these increments, the development process becomes cheaper and faster as a result of less time spent reworking the system.\cite{sommervilleSoftwareEngineering2016}
-This section will discuss how we are using an agile approach for our project, focusing specifically on the Scrum framework.
+By continuously getting feedback and working with these increments, the development process becomes cheaper and faster as less time is spent adapting to changes.\cite{sommervilleSoftwareEngineering2016}
+This section will discuss how an agile approach has been utilized in this project. 
 
-\subsection*{Scrum Framework}
-Agile is a set of values, and does not present practical activities which a process model would consist of\cite{sutherlandScrumArtDoing2014}.
-Therefore, the team has decided on Scrum\cite{scrumdotorg}, which is an agile framework that puts the values to practice.
-This decision was made due to prior experience with using the framework amongst team members.
+\subsection{Scrum Framework}
+Agile is a set of values and does not present practical activities which a process model would consist of{sutherlandScrumArtDoing2014}.
+Therefore, the team has decided on the agile framework Scrum\cite{scrumdotorg}. Scrum chosen as it puts the values to practice and due to prior experience with using the framework amongst team members. \todo{What does it mean to put the values to practice}
 
-Scrum introduces a variety of new practices and roles, which we will describe in the following sections.
+Scrum introduces a number of practices and roles, which is described in the following sections. 
 
 \subsubsection*{Roles and Responsibilities}
-In a Scrum team, there is a Scrum Master, a Product Owner, and developers.
+A Scrum team consists of three roles: the Scrum Master, a Product Owner, and developers.
```

Comment:
Not sure leading with a colon is great. In general, using it to list things in this kind of report makes the writing seem lazier. <- IMO